src/app/shared/axiosHelper.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
       headers: {
         "Content-Type": "application/x-www-form-urlencoded",
         "ipv-session-id": req.session.ipvSessionId,
+        "client-oauth-session-id": req?.session?.clientOauthSessionId,
```

Comment:
Would it be reasonable to create a new generateAxiosConfig for this specific event otherwise an undefined client-oauth-session-id will always be passed around everytime we are doing a post